crate_universe/extension.bzl

```diff
@@ -290,10 +325,39 @@ _annotation = tag_class(
     ),
 )
 
+_from_specs = tag_class(
+    doc = "Generates a repo @crates from a Cargo.toml / Cargo.lock pair",
```

Comment:
Nit: This looks like it needs to be updated

Reply:
yep good catch, ty!